Patches/[O21] Outer Rim/Patch_OuterRim_SandCrawler.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Outer Rim - Tatooine Sandcrawler (Continued)</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+		
+			<li Class="PatchOperationAdd">
+				<xpath>Defs</xpath>
+				<value>
+					<CombatExtended.BodyShapeDef>
```

Comment:
An interesting addition, but why not just use the Vehicle shape instead?

Reply:
The reason I added this is to make pawns shoot at the sandcrawler instead of above it. In the end it didn't work.